web/src/inbox_ui.ts

```diff
@@ -821,6 +903,83 @@ function reset_data(): {
     topics_dict = get_sorted_stream_topic_dict();
     const is_dms_collapsed = collapsed_containers.has("inbox-dm-header");
 
+    for (const [, stream_data] of streams_dict) {
```

Comment:
`stream_data` seems like a bad variable, given we have a module by the same name. Maybe just `stream_context`, to match the type, would be better?